lib/logux.rb

```diff
@@ -55,6 +55,16 @@ def self.add(type, meta: {})
     logux_add.call(type, meta: logux_meta)
   end
 
+  def self.undo(meta = Logux::Meta.new({}), reason = nil)
+    logux_add = Logux::Add.new
+    logux_meta = Logux::Meta.new(meta.merge(status: 'processed'))
```

Comment:
@markfrst I asked @ai about it, and we don't need `processed` here, so let's remove this string

Reply:
okay